pkg/actors/internal_actor.go

```diff
@@ -174,3 +183,11 @@ func DecodeInternalActorData(data []byte, e any) error {
 	}
 	return nil
 }
+
+// DecodeInternalActorReminderData decodes internal actor reminder data payloads and stores the result in e.
+func DecodeInternalActorReminderData(data []byte, e any) error {
```

Comment:
nit: It's not clear why this method exists - do we intend to add extra data or functionality to it? If not why not just `json.Unmarshal` where this gets called?